sites/fast-website/src/app/components/card-section/card-section.styles.ts

```diff
@@ -6,7 +6,7 @@ export const CardSectionStyles = css`
         --flow: column;
     }
 
-    @media screen and (max-width: 1000px) {
+    @media screen and (max-width: 900px) {
```

Comment:
It might good to start storing these breakpoints in constants file so we can use them elsewhere. We could follow a similar model that we do with the react version of the `BreakpointTracker` which is store and array of numbers and then reference them throughout the app. This case we can just use `string[]`.

```
export siteBreakpoints: string[] = [0, 900px, 1200px];
```

```
@media screen and (max-width: siteBreakpoints[1]) {}
```

Reply:
That sounds great.  I'll get on that.  Thanks Jeff.

Reply:
Done